tests/test_smartcar.py

```diff
@@ -57,7 +64,11 @@ def test_get_auth_url(self):
             'state': 'stuff'
         })
         expected = smartcar.const.CONNECT_URL + '/oauth/authorize?' + query
-        self.assertEqual(actual, expected)
+        
+        expected_params = parse_qs(expected)
+        actual_params = parse_qs(actual)
+
+        assertDeepEquals(self, expected_params, actual_params)
```

Comment:
just a question for curiosity! Why are you using parse_qs instead of just comparing the URLs generated?

Reply:
I don't want to accidentally test for ordering within the query string, as this might change when the code changes in the future. By using`parse_qs` and comparing the dictionaries with `assertDeepEquals()`, the test doesn't care about any future ordering changes. :smile:

Reply:
that makes a lot of sense!! thanks for the explanation :100: :100: